README.md

```diff
@@ -224,7 +224,7 @@ mindex.updateRecord({
 
 | key | default | type | description |
 | --- | --- | --- | --- |
-| keyList | undefined | Mixed | An array of values to retrieve from the index. If the index is not compound a string or integer may be used. |
+| keyList | undefined | Mixed | An array of values to retrieve from the index. If the index is not a compound object, a string or integer may be used. |
```

Comment:
Sure not a problem.
On 29 Mar 2016 18:48, "Bryan Morris" <notifications@github.com> wrote:

> In README.md
> <https://github.com/internalfx/mindex/pull/1#discussion_r57757451>:
>
> > @@ -224,7 +224,7 @@ mindex.updateRecord({
> >
> >  | key | default | type | description |
> >  | --- | --- | --- | --- |
> > -| keyList | undefined | Mixed | An array of values to retrieve from the index. If the index is not compound a string or integer may be used. |
> > +| keyList | undefined | Mixed | An array of values to retrieve from the index. If the index is not a compound object, a string or integer may be used. |
>
> The index can either be compound or not. I would not say that the index
> itself is a compound object.
>
> The index being compound means that it is indexed by multiple keys.
>
> Can you remove this change from your PR?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/internalfx/mindex/pull/1/files/2f740c0cb5f56c390b375d3680671ef3febc9501#r57757451>
>